docs/reference/modules/settings.md

```diff
@@ -0,0 +1,127 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/settings`
+
+**Alias:** `apos.settings`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+This module governs the Personal Settings menu that allows users to change their preferences. Apostrophe projects can configure this menu through `@apostrophecms/settings`.
```

Comment:
As such, these settings are **personal** to the user. If you're looking for sitewide settings that govern overall site behavior, check out `@apostrophecms/global`.

Reply:
Changed.